index.html

```diff
@@ -304,7 +304,7 @@ <h2>Skills</h2>
     <section class="contact-section">
       <div class="contact-container">
         <div class="contact-div">
-          <h1>Contact me</h1>
+          <h2>Contact me</h2>
```

Comment:
- [ ] Great job working on the headings of your page! 💯 In order to improve further the accessibility of your page you could modify the labels of your contact form as explained below.

Following the [Easy Checks](https://www.w3.org/WAI/test-evaluate/preliminary/#forms) accessibility instructions for forms:

![image](https://user-images.githubusercontent.com/77069011/207383718-17a0b676-3bf8-43c4-ac44-2f2852e26514.png)

Please add an indicator of which fields are required! 🤓 🛠️